addons/web/static/src/views/kanban/kanban_dashboard.scss

```diff
@@ -202,7 +206,10 @@
     }
 
     .o_favorite {
-        @include o-position-absolute(3px, $left: 0);
+        @include o-position-absolute(
+            $top: var(--KanbanRecord-padding-v),
```

Comment:
We need to keep the value of `3px` or replace it with a variable that has this value, otherwhise the alignment of the icon won't be right.
<img width="347" alt="Capture d’écran 2022-08-05 à 11 03 35" src="https://user-images.githubusercontent.com/80679690/183044210-d1fd4ede-5228-400c-ad44-d116dc892654.png">

Reply:
I've check that and seems to work fine... let's hope I was wrong and we don't have to deal with (yet another) template discrepancy .

Reply:
Looks fine locally and on runbot :/
Old build?

<img width="348" alt="image" src="https://user-images.githubusercontent.com/8558351/183060162-40c86e3f-13a5-455a-83a7-ed5c4935b305.png">